File: chapter1-Απάντηση/gp5.rb
require 'time'

puts "name ?"

name = gets

hour = Time.new.hour

if hour<14
    print("Καλημέρα ", name)
else
    print("Καλησπέρα ", name)
end
    
wait = 3
sleep(wait)

answer=42
print("Η απάντηση είναι ...", answer)

File: chapter1/gp5.py
import time

name = input("Πώς σε λένες; ")

hour = time.localtime().tm_hour

if hour<14:
    print("Καλημέρα", name)
else:
    print("Καλησπέρα", name)
    
wait = 3
time.sleep(wait)

answer=42
print("Η απάντηση είναι ...", answer)

